src/Microsoft.ML.Ensemble/Microsoft.ML.Ensemble.csproj

```diff
@@ -10,6 +10,7 @@
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
     <ProjectReference Include="..\Microsoft.ML.StandardLearners\Microsoft.ML.StandardLearners.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Transforms\Microsoft.ML.Transforms.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.FastTree\Microsoft.ML.FastTree.csproj" />
```

Comment:
><ProjectReference Include="..\Microsoft.ML.FastTree\Microsoft.ML.FastTree.csproj" /> [](start = 4, length = 84)

We need this due to the usage of FastTree as the default traniner for some ensemble classes. This could be fine, but I wonder if using something from `StandardLearners` might make a bit more sense, if we envision that someday FastTree will be factored into a separate nuget package. (Maybe we don't want that though, in which case this dependency is probably fine.)

Reply:
One advantage to this change is that it brings to light these dependencies. Before we had a weakly-typed dependency using strings and DI.  So if we ever did refactor FastTree to be separate, we could have easily broken this dependency.

I've logged #682 for this issue.

Reply:
Thanks @eerhardt ... yes I expect we'll see some more of this "hidden dependency" becoming exposed as we move away from direct usage of subcomponents and dependency injection. I recall a similar issue in #446 with the normalizers being *defined* in `Microsoft.ML.Transforms` and *used* in `Microsoft.ML.Data`, despite the former project depending on the latter. 😄